### PR TITLE
New: Add Splunk as customer

### PR DESCRIPTION
**Pull request by bvandewalle**
Adding Splunk as a customer

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by kebe7jun** [comment]
It seems that the order of you company is wrong.   @bvandewalle

File: content/en/about/community/customers/index.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Here are just a few of the organizations we know are using Istio. If you want to
     {{< company_logo link="https://www.skyscanner.com" logo="./skyscanner.svg" alt="Skyscanner" >}}
     {{< company_logo link="https://www.softonic.com" logo="./softonic.png" alt="Softonic S.L." >}}
     {{< company_logo link="https://www.spotify.com" logo="./spotify.svg" alt="Spotify" >}}
+    {{< company_logo link="https://www.splunk.com" logo="./splunk.png" alt="Splunk" >}}
```
Comment (kebe7jun) [inline content/en/about/community/customers/index.md:64]: It seems that the order of you company is wrong.

Comment (bvandewalle) [reply content/en/about/community/customers/index.md:63]: Ah good catch, thanks!

**Review by kebe7jun — APPROVED** [review]

**Review by craigbox — APPROVED** [review]
Thank you for using Istio!

**Pull request merged**